<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>161</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignTop">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>1. Image Processing</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QPushButton" name="pushButton_1_1">
       <property name="text">
        <string>1.1 Load Image</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QPushButton" name="pushButton_1_2">
       <property name="text">
        <string>1.2 Color Seperation</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QPushButton" name="pushButton_1_3">
       <property name="text">
        <string>1.3 Color Transformation</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QPushButton" name="pushButton_1_4">
       <property name="text">
        <string>1.4 Blending</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>160</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item alignment="Qt::AlignTop">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>2. Image Smoothing</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QPushButton" name="pushButton_2_1">
       <property name="text">
        <string>2.1 Gaussian Blur</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QPushButton" name="pushButton_2_2">
       <property name="text">
        <string>2.2 Bilateral Filter</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QPushButton" name="pushButton_2_3">
       <property name="text">
        <string>2.3 Median Filter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>160</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item alignment="Qt::AlignTop">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>3. Edge Detection</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QPushButton" name="pushButton_3_1">
       <property name="text">
        <string>3.1 Gaussian Blur</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QPushButton" name="pushButton_3_2">
       <property name="text">
        <string>3.2 Sobel X</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QPushButton" name="pushButton_3_3">
       <property name="text">
        <string>3.3 Sobel Y</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QPushButton" name="pushButton_3_4">
       <property name="text">
        <string>3.4 Magnitude</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>30</y>
      <width>160</width>
      <height>521</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item alignment="Qt::AlignTop">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>4. Transforms</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QPushButton" name="pushButton_4_1">
       <property name="text">
        <string>4.1 Resize</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QPushButton" name="pushButton_4_2">
       <property name="text">
        <string>4.2 Translation</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QPushButton" name="pushButton_4_3">
       <property name="text">
        <string>4.3 Rotation, Scaling</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignTop">
      <widget class="QPushButton" name="pushButton_4_4">
       <property name="text">
        <string>4.4 Shearing</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
